controller_manager/test/test_spawner_unspawner.cpp

```diff
@@ -327,6 +327,61 @@ TEST_F(TestLoadController, unload_on_kill)
   ASSERT_EQ(cm_->get_loaded_controllers().size(), 0ul);
 }
 
+TEST_F(TestLoadController, spawner_test_fallback_controllers)
```

Comment:
Why was this moved?

Reply:
I moved it for better log tracking as the newly added test is placed in between an original test. 

https://github.com/ros-controls/ros2_control/blob/9d1aff96291302c6db153f863792350e352f66a9/controller_manager/test/test_spawner_unspawner.cpp#L330-L407